BeardedSpice/BSTrack.m

```diff
@@ -18,19 +18,21 @@
 NSString *const kBSTrackNameFavorited = @"favorited";
 NSString *const kBSTrackNameIdentifier = @"BSTrack Notification";
 
+const NSString *kImageLock = @"kImageLock";
+
 @interface BSTrack ()
 
 @property (nonatomic, strong) NSString *imageURL;
 
-@property (nonatomic, strong) NSString *lastImageUrlString;
-@property (nonatomic, strong) NSImage *lastImage;
-
 - (void)_loadImage;
 
 @end
 
 @implementation BSTrack
 
+static NSString *_lastImageUrlString;
+static NSImage *_lastImage;
```

Comment:
Please leave these as instance variables. This makes them singleton values that aren't properly ARC controlled and can have stale or invalid content that crosses over between BSTrack and tab instances.